src/lib/core/components/visualizations/InputVariables.tsx

```diff
@@ -13,10 +13,13 @@ import {
   mapStructure,
   preorder,
 } from '@veupathdb/wdk-client/lib/Utils/TreeUtils';
+import Switch from '@veupathdb/components/lib/components/widgets/Switch';
+import { makeEntityDisplayName } from '../../utils/study-metadata';
 
 interface InputSpec {
   name: string;
   label: string;
+  role?: 'primary' | 'stratification'; // TO DO: make mandatory when all viz are updated??
```

Comment:
Can this be required now?

Reply:
Ooh yes, good catch.  I just had to update MosaicViz